### DISCUSSION TITLE
ASMedia Firmware collection

### DISCUSSION THREAD

**Discussion by smx-smx** [smx-smx/ASMTool#8] [General]
Post here the firmware dump obtained with `ASMTool flash_read`. together with the details about your motherboard and chip (PCI IDs, chip name)

**Comment by WilltoBe2** [comment]
PCI-e USB 3.2 Card 20 Gbps Gen 2 X4 with 5 Ports (USB-A -USB-C) Super-speed Splitter Adapter for Windows
Model: ULS UP10052
(Above is not my own description)

Chipset: asmedia ASM3142
ID's: 1B21:2142

![asmedia_ASM3142](https://github.com/smx-smx/ASMTool/assets/139520028/90b90583-f903-4659-8f62-16c058dc148b)
![Back](https://github.com/smx-smx/ASMTool/assets/139520028/32ecfc04-c3e8-42bc-8ec8-d7ab24097cb3)
![Connectors](https://github.com/smx-smx/ASMTool/assets/139520028/f85cb2b8-741a-47e9-94c1-c49bac15fc5e)
![Front](https://github.com/smx-smx/ASMTool/assets/139520028/64696363-9706-4994-909e-d82268f50755)
![Front_Closer](https://github.com/smx-smx/ASMTool/assets/139520028/cda563f5-7ce6-49b5-86e9-f4532b78cc67)
![VLI_VL822-Q8](https://github.com/smx-smx/ASMTool/assets/139520028/92e4a56e-de17-473e-b9ad-2592fc9f69c2)
[dump_files.zip](https://github.com/smx-smx/ASMTool/files/12047458/dump_files.zip)

**Comment by c4801725870** [comment]
[GA-X299-UD4-PRO-1_0.zip](https://github.com/user-attachments/files/17967906/GA-X299-UD4-PRO-1_0.zip)
Motherboard: X299 UD4 PRO 1.0
Chip: ASM3142
SVID:  1b21
SSID: 2142

ASM3142_170308_70_02_00.bin: produce using flash_read
ASM3142_MEM_170308_70_02_00.bin: produce using mem_read
